calculator/test_unittest_calculator.py

```diff
@@ -0,0 +1,40 @@
+"""Unit test module for testing calculator module."""
+
+import unittest
+
+from calculator import evaluate
+
+
+class TestOperations(unittest.TestCase):
+    """Test the supported operations by the calculator."""
+    def test_basic(self):
+        """Test the basic operations."""
+        self.assertEqual(evaluate("1 + (2 + 3)"), 6.0)
+        self.assertEqual(evaluate("(8 - 4) - 6"), -2.0)
+
+        self.assertEqual(evaluate("3* -5"), -15.0)
+        self.assertEqual(evaluate("65 /5"), 13.0)
```

Comment:
why did you decide those are basic operations?
What if I asked that + sign do something completely different that adding, then it will not be basic?